src/routes/email/email.controller.delete-email.spec.ts

```diff
@@ -74,16 +74,20 @@ describe('Email controller delete email tests', () => {
     await app.close();
   });
 
-  it('deletes email successfully', async () => {
+  it.each([
+    // non-checksummed address
+    { safeAddress: faker.finance.ethereumAddress() },
```

Comment:
Nit: what do you think about calling `toLowerCase` to be explicit? (And in case it ever changes to return a checksummed address.)

Reply:
Makes total sense. Added [1ad54f7](https://github.com/safe-global/safe-client-gateway/pull/1218/commits/1ad54f722841490e3d93ca861b0e730021783d19).